{"ygPerms":{"resourceCapabilityList":[{"resourceType":"GROUP","capabilities":[{"name":"READ"},{"name":"JOIN"}]},{"resourceType":"PHOTO","capabilities":[]},{"resourceType":"FILE","capabilities":[]},{"resourceType":"MEMBER","capabilities":[]},{"resourceType":"LINK","capabilities":[]},{"resourceType":"CALENDAR","capabilities":[]},{"resourceType":"DATABASE","capabilities":[]},{"resourceType":"POLL","capabilities":[]},{"resourceType":"MESSAGE","capabilities":[{"name":"READ"}]},{"resourceType":"PENDING_MESSAGE","capabilities":[]},{"resourceType":"ATTACHMENTS","capabilities":[{"name":"READ"}]},{"resourceType":"PHOTOMATIC_ALBUMS","capabilities":[]},{"resourceType":"MEMBERSHIP_TYPE","capabilities":[]},{"resourceType":"POST","capabilities":[{"name":"READ"}]},{"resourceType":"PIN","capabilities":[]}],"groupUrl":"groups.yahoo.com","intlCode":"us"},"comscore":"pageview_candidate","ygData":{"userId":44728387,"authorName":"David Goodwin","from":"David Goodwin &lt;dgoodwin@...&gt;","profile":"dggoodwin","replyTo":"LIST","senderId":"6cipkYo6DGsg6CJg65Q8nGy4cJ7wdxztOU1VJZVC966DRUhVg5M6TBwcXqRWm3Qsdvfl7UUqAIL8X6oo39wBpq15uoDm5APioPYE","spamInfo":{"isSpam":false,"reason":"12"},"subject":"Re: [cantera] ct2ctml problems?  (May not be a Cantera problem, though)","postDate":"1115781993","msgId":786,"canDelete":false,"contentTrasformed":false,"systemMessage":false,"headers":{"messageIdInHeader":"PGNjZDhiMzMxYjU3ZjM1ZmY2YjE5ZTQyZDFmZDNkMDA1QGNhbHRlY2guZWR1Pg==","inReplyToHeader":"PGQ1cDV2Zys5amtkQGVHcm91cHMuY29tPg==","referencesHeader":"PGQ1cDV2Zys5amtkQGVHcm91cHMuY29tPg=="},"prevInTopic":785,"nextInTopic":787,"prevInTime":785,"nextInTime":787,"topicId":785,"numMessagesInTopic":4,"msgSnippet":"Mark, I suspect the problem is that the Cantera Python extension module was built to link to one version of the Python DLL, and can t be used with a different","rawEmail":"Return-Path: &lt;dgoodwin@...&gt;\r\nX-Sender: dgoodwin@...\r\nX-Apparently-To: cantera@yahoogroups.com\r\nReceived: (qmail 40744 invoked from network); 11 May 2005 03:27:04 -0000\r\nReceived: from unknown (66.218.66.218)\n  by m23.grp.scd.yahoo.com with QMQP; 11 May 2005 03:27:04 -0000\r\nReceived: from unknown (HELO outgoing-mail.its.caltech.edu) (131.215.239.19)\n  by mta3.grp.scd.yahoo.com with SMTP; 11 May 2005 03:27:04 -0000\r\nReceived: from water-dog (water-dog [192.168.1.26])\n\tby water-ox-postvirus (Postfix) with ESMTP id 5796426AC23\n\tfor &lt;cantera@yahoogroups.com&gt;; Tue, 10 May 2005 20:26:59 -0700 (PDT)\r\nReceived: from water-ox ([192.168.1.10])\n\tby water-dog (MailMonitor for SMTP v1.2.2 ) ;\n\tTue, 10 May 2005 20:26:58 -0700 (PDT)\r\nReceived: from [216.216.49.68] (unknown [216.216.49.68])\n\tby water-ox.its.caltech.edu (Postfix) with ESMTP id A91CF26AC4E\n\tfor &lt;cantera@yahoogroups.com&gt;; Tue, 10 May 2005 20:26:49 -0700 (PDT)\r\nMime-Version: 1.0 (Apple Message framework v622)\r\nIn-Reply-To: &lt;d5p5vg+9jkd@...&gt;\r\nReferences: &lt;d5p5vg+9jkd@...&gt;\r\nContent-Type: text/plain; charset=US-ASCII; format=flowed\r\nMessage-Id: &lt;ccd8b331b57f35ff6b19e42d1fd3d005@...&gt;\r\nContent-Transfer-Encoding: 7bit\r\nDate: Tue, 10 May 2005 22:26:33 -0500\r\nTo: cantera@yahoogroups.com\r\nX-Mailer: Apple Mail (2.622)\r\nX-Spam-Status: No, hits=0.0 tagged_above=-100000.0 required=5.0 \r\nX-Spam-Level: \r\nX-eGroups-Msg-Info: 1:12:0\r\nFrom: David Goodwin &lt;dgoodwin@...&gt;\r\nSubject: Re: [cantera] ct2ctml problems?  (May not be a Cantera problem, though)\r\nX-Yahoo-Group-Post: member; u=44728387\r\nX-Yahoo-Profile: dggoodwin\r\n\r\nMark,\n\nI suspect the problem is that the Cantera Python extension module was \nbuilt to link to one version of the Python DLL, and can&#39;t be used with \na different version.  Try running the temporary Python file that \nct2ctml creates, and see what error message it gives you.  If this is \nthe problem, you can build it from the source with whatever Python \nversion you like.\n\nDave\n\n\n\n\nOn May 9, 2005, at 9:23 PM, markscha wrote:\n\n&gt; Well, just when I was getting productive, I ran into a problem with\n&gt; Cantera/Python.\n&gt;\n&gt; I had Cantera working on this XP workstation, but I was trying to\n&gt; install another Python package (gnuplot-py 1.7) when I realized that\n&gt; I still had an older version of Python installed (2.3), and this was\n&gt; interfering with the gnuplot-py package.  So, I uninstalled the 2.3\n&gt; version, removed it from my PATH, and viola---Cantera started\n&gt; complaining:  &quot;...could not convert input file to CTML&quot;.\n&gt;\n&gt; I uninstalled Cantera, Python, Numpy, etc., reboot, re-installed\n&gt; Python, Numpy stuff, and Cantera, and I still get the same thing.\n&gt;\n&gt; I am using Cantera 1.5.5 and Python 2.4, etc. (the standard stuff)\n&gt; on a XP workstation.\n&gt;\n&gt; So, am I screwed or what?  I have tried to clean my system several\n&gt; times, and I keep getting the same behavior.  I have verified the\n&gt; file pops up in the correct TMP directory.  I have also tried\n&gt; replaciing the PYTHON_CMD and put c:&#92;Python24 in the PATH.  I have\n&gt; left the directories to be the default values, and I have tried some\n&gt; non-defaults.\n&gt;\n&gt; Any thoughts?\n&gt;\n&gt; -----Mark\n&gt;\n&gt; An example Python Traceback follows:\n&gt;\n&gt; Traceback (most recent call last):\n&gt;   File &quot;C:&#92;Documents and Settings&#92;mas&#92;Desktop&#92;Desktop&#92;Cantera2\n&gt; &#92;cantera15&#92;demos&#92;Python&#92;mix1.py&quot;, line 25, in ?\n&gt;     gas_a = Air()\n&gt;   File &quot;C:&#92;Python24&#92;Lib&#92;site-packages&#92;Cantera&#92;gases.py&quot;, line 40, in\n&gt; Air\n&gt;     return Solution(src=&quot;air.cti&quot;, id=&quot;air&quot;)\n&gt;   File &quot;C:&#92;Python24&#92;Lib&#92;site-packages&#92;Cantera&#92;solution.py&quot;, line 38,\n&gt; in __init__\n&gt;     root = XML.XML_Node(name = &#39;doc&#39;, src = src, preprocess = 1)\n&gt;   File &quot;C:&#92;Python24&#92;Lib&#92;site-packages&#92;Cantera&#92;XML.py&quot;, line 35, in\n&gt; __init__\n&gt;     self._xml_id = _cantera.xml_get_XML_File(src)\n&gt; error:\n&gt;\n&gt; ************************************************\n&gt;                 Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: ct2ctml\n&gt; Error:   could not convert input file to CTML\n&gt;  command line was: c:&#92;python24&#92;python.exe C:&#92;DOCUME~1&#92;mas&#92;LOCALS~1\n&gt; &#92;Temp/.cttmp1115691081.pyw&gt; ct2ctml.log 2&gt;&1\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt; Yahoo! Groups Links\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n\n\n\n"}}